lib/Listener/BotListener.php

```diff
@@ -65,6 +67,14 @@ public function handle(Event $event): void {
 			$this->botService->afterChatMessageSent($event, $messageParser);
 			return;
 		}
+		if ($event instanceof ReactionAddedEvent) {
+			$this->botService->afterReaction($event, $messageParser);
+			return;
+		}
+		if ($event instanceof ReactionRemovedEvent) {
+			$this->botService->undoReaction($event, $messageParser);
```

Comment:
Just a feeling, should that be `afterReactionAdded` and `afterReactionRemoved` instead?

Reply:
I got mixed up from the ActivityPub wording, but yeah makes sense.